### PR TITLE
Automating deb package updates via GitHub Actions

### LINKED ISSUE #537: Suggestion: better version and update management
When going through the code, I've noticed that package manager is not always selecting the most up to date version with respect to the chosen snapshot. Currently this seems to be solved by pinning the version of specific libraries in the WORKSPACE file, but this looks a bit error prone as an approach.

My proposal:
 * Introduce a versions file per package bundle (deb9 and deb10) to clearly see what versions are being used. For example: https://github.com/petermylemans/distroless/blob/versions/package_bundle_debian10.versions
 * Introduce a better version selection algorithm based on https://github.com/theclimatecorporation/python-dpkg 

To fully complete the solution, I would suggest to make a kind of bot/github action that takes the latest snapshot from debian 9 / 10, regenerate the version files and see if there is a difference. If so, the bot can send a PR for review.

### PR DESCRIPTION
**Pull request by chanseokoh**
Does this look good @petermylemans?

Fixes #537.

### CODE REVIEW

**Comment by petermylemans** [comment]
Looks good to me!

You might want to consider renaming the file from main.yml to something more specific like e.g. update-snapshots.yml or update-debs.yml.

File: .github/workflows/update-deb-package-snapshots.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: update-snapshots
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```
Comment (briandealwis) [inline .github/workflows/update-deb-package-snapshots.yml:4]: Might be better to schedule this slightly off-peak?  Help Github avoid computation cascades at midnight?

File: .github/workflows/update-deb-package-snapshots.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      uses: abhinavsingh/setup-bazel@v3
+      with:
+        version: 3.2.0
+
+    - name: Run update script
+      run: ./updateWorkspaceSnapshots.sh
+    
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v2
+      with:
```
Comment (briandealwis) [inline .github/workflows/update-deb-package-snapshots.yml:22]: Do you want to set an author and committer? https://github.com/peter-evans/create-pull-request#committer-and-author

File: .github/workflows/update-deb-package-snapshots.yml (hunk 1 above)
Comment (chanseokoh) [reply .github/workflows/update-deb-package-snapshots.yml]: Good idea. Going with UTC 8:30am (EST 4:30 and 1:30 PST as of now).

File: .github/workflows/update-deb-package-snapshots.yml (hunk 2 above)
Comment (chanseokoh) [reply .github/workflows/update-deb-package-snapshots.yml:22]: Oh, that's interesting. But I think it makes sense to have the author as the bot.

**Pull request merged**